idp/wechat.go

```diff
@@ -249,6 +250,20 @@ func BuildWechatOpenIdKey(appId string) string {
 	return fmt.Sprintf("wechat_openid_%s", appId)
 }
 
+func BuildNickName(username, id string) string {
```

Comment:
应该是 `BuildUserName`？

```suggestion
func BuildUserName(nickname, id string) string {
```